base/stats.go

```diff
@@ -383,7 +383,7 @@ type CacheStats struct {
 	NonMobileIgnoredCount *SgwIntStat `json:"non_mobile_ignored_count"`
 	// The total number of active channels.
 	NumActiveChannels *SgwIntStat `json:"num_active_channels"`
-	// The total number of skipped sequences. This is a cumulative value.
+	// The total number of skipped sequences.
```

Comment:
Isn't this still a cumulative value?  I don't think this is a big deal but I think this could stay.